### PR TITLE
WIP: Fix issue 13376

### PR DESCRIPTION
**Pull request by SamMousa**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | maybe
| Fixed issues  | #13376

### CODE REVIEW

**Comment by SamMousa** [comment]
Hmm seems I based this off the wrong branch.

File: framework/web/Request.php [hunk 1]
```diff
@@ -961,7 +965,7 @@ public function getServerPort()
      */
     public function getReferrer()
     {
-        return isset($_SERVER['HTTP_REFERER']) ? $_SERVER['HTTP_REFERER'] : null;
+        $this->headers->get('Referer');
```
Comment (lynicidn) [inline framework/web/Request.php]: return

Comment (SamMousa) [inline framework/web/Request.php]: :O Does that mean you guys don't test this? ;-)

Comment (samdark) [inline framework/web/Request.php]: Yes. Seems this part somehow slipped through...

**Pull request closed without merging**